todo.sh

```diff
@@ -45,6 +45,7 @@ shorthelp()
 		    add|a "THING I NEED TO DO +project @context"
 		    addm "THINGS I NEED TO DO
 		          MORE THINGS I NEED TO DO"
+		    addp PRIORITY "TEXT TO ADD"
```

Comment:
Given that we have `add|a` and `pri|p`, I'd expect the new command to be available as `addpri|ap`; `addp` is an inconsistent mixture of long and short forms.